src/main/java/org/softserve/edu/utils/ConverterDTO.java

```diff
@@ -0,0 +1,58 @@
+package org.softserve.edu.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.softserve.edu.models.environmentJSON.Environment;
+
+import java.io.File;
+import java.io.IOException;
+
+
+public class ConverterDTO {
```

Comment:
This class should contain only static methods:
1. String dtoToJsonString(Object dtoClass){return String}
2. String dtoToPrettyJsonString(Object dtoClass) {return String} // not required but it would be good to have
3. convertStringToObject()
public static <T> T convertStringToObject(Class<T> dtoClass, String content, boolean failOnUnknown) {
      ObjectMapper  dtoObjectMapper = new ObjectMapper ();
        dtoObjectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, failOnUnknown);
        try {
            return dtoObjectMapper.readValue(content, dtoClass);
        } catch (IOException e) {
            LOGGER.error(e);
            return null;
        }
    }

Example of using: ConverterDTO .convertStringToObject(Environment.class, fileContent, false);
// Environment class rename to UsedEnvironment
Please add these methods and test all of them

Reply:
All 3 methods implemented